mapdata/WarcraftLegacies/AbilityData/Skin/1194537281.json

```diff
@@ -0,0 +1,115 @@
+{
+  "OldId": 1835619649,
+  "NewId": 1194537281,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 1718840929,
+      "Type": 3,
+      "Value": "(Druid - Tortolla)"
+    },
+    {
+      "Id": 2037082209,
+      "Type": 3,
+      "Value": "W"
+    },
+    {
+      "Id": 1802007137,
+      "Type": 3,
+      "Value": "W"
+    },
+    {
+      "Id": 1952805473,
+      "Type": 3,
+      "Value": "Learn Torrent - [|cffffcc00Level %d|r]"
+    },
+    {
+      "Id": 1953854049,
+      "Type": 3,
+      "Value": "The hero creates a powerful torrent out of the earth, stunning and damaging enemies in a large area. |n|n|cffffcc00Level 1|r - \u003CAUim,DataC1\u003E damage, 1 second stun. |n|cffffcc00Level 2|r - \u003CAUim,DataC2\u003E damage, 1.5(1.25) second stun. |n|cffffcc00Level 3|r - \u003CAUim,DataC3\u003E damage, 2(1.5) second stun. |n|cffffcc00Level 4|r - \u003CAUim,DataC4\u003E damage, 2.5(1.75) second stun."
```

Comment:
It's not really a "large area", it's a line like Impale, so the tooltip should reflect that.

Reply:
actually the area of effect is almost twice as large as impale, hence why i added large here

Reply:
As discussed on Discord, it's still done in a line so it needs to indicate that somehow.